amida-tech/orange-api
# Feature/orange 98 go activities

***Merge [PR 72](https://github.com/amida-tech/orange-api/pull/72) first as this is branched off that***.

# What does this do
This adds two fields to journal entries:
* `activity`: a string to represent an activity
* `activityMinutes`: the number of minutes the activity was performed

Either both must be set or neither can be set.
# How to test
Use postman to create journal entries http://amida-tech.github.io/orange-api/#journal-entries
Should get
* success on journal entry with neither field
* success on journal entry with both fields, `activityMinutes` >= 1
* failure on entry with only one of the two fields
* failure on entry with both fields, `activityMinutes` < 1
